ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift

```diff
@@ -283,15 +285,28 @@ extension KeymanWebViewController {
     do {
       data = try JSONSerialization.data(withJSONObject: stub, options: [])
     } catch {
-      log.error("Failed to serialize keyboard stub: \(error)")
+      let event = Sentry.Event(error: error)
+      event.message = SentryMessage(formatted: "Failed to serialize keyboard stub: \(error)")
+      event.extra = [:]
+      event.extra!["id"] = stub["KI"]
+      event.extra!["package"] = stub["KP"]
```

Comment:
Can stub["KP"] be undefined?

Reply:
`undefined` does not exist in Swift.  The worst case is `nil`, and that wouldn't break this.